### PR TITLE
planning/ompl depends on ODE

### PR DESCRIPTION
**Pull request by chhtz**
Missing dependency

### CODE REVIEW

File: manifests/planning/ompl.xml [hunk 1]
```diff
@@ -10,4 +10,5 @@
   <url>http://ompl.kavrakilab.org</url>
   <depend package="boost" />
   <depend package="flann" optional="1" />
+  <depend package="simulation/ode" />
```
Comment (doudou) [inline manifests/planning/ompl.xml]: Unless I am mistaken, this adds a dependency of the rock package set to the rock.simulation package set.

I would suggest adding a simulation/ode osdep in the rock package set, and fixing the add_osdeps_override line in rock.simulation (which is currently completely bogus).

**Comment by chhtz** [comment]
Sorry, I did not check that ODE is actually an OS-dependency in rock-core.
I guess just adding `<osdep name='ode' />` here and changing the first argument of `add_osdeps_override` in rock.simulation from `external/ode` to just `ode` would work as well?
Or is it better style to name the osdep `simulation/ode` instead of just `ode`?

**Comment by chhtz** [comment]
Hm, just adding `<osdep name='ode'>` does not seem to work (I'm not sure how it is supposed to work). Also, as there is no mention of ODE in the CMakeLists.txt of ompl, I guess this will fail if ODE is not installed in one of the default install directories (that is not a use case for me, so I don't mind).
And overall, this is not urgent for me, I just noticed this after a `rm -rf install; amake`

**Comment by chhtz** [comment]
I did not mean to close this -- I just accidentally overwrote my fork (but the solution was not ideal anyway). I'll re-open this as issue.

**Pull request closed without merging**